onnx/defs/schema.h

```diff
@@ -818,6 +818,10 @@ class OpSchema final {
     return tensor_inference_function_ ? true : false;
   }
 
+  InferenceFunction get_type_and_shape_inference_function() const {
```

Comment:
I think this getter function already exists: see `GetTypeAndShapeInferenceFunction`.

Reply:
You're right, I think I must have thought it was private for some reason. Fixed!